src/main/java/Duke.java

```diff
@@ -49,52 +49,118 @@ public static void main(String[] args) {
             // Read inputs
             System.out.println("\tCall out a smurf to do a job for you!");
             Scanner in = new Scanner(System.in);
-            String command;
-            command = in.nextLine();
+            String userInput;
```

Comment:
Perhaps this variable name could be in plural?

Reply:
Thanks for pointing this out. I thought about it, everytime we pass through the loop, we are only taking 1 input. The whole string is considered userInput. Hence the singular form.